networking/multiple_networks/configuring-multi-network-policy.adoc

```diff
@@ -6,14 +6,22 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you can configure network policy for additional networks.
+As a cluster administrator, you can configure multi-network for additional networks. You can specify multi-network policy for SR-IOV and macvlan additional networks. Macvlan additional networks are fully supported other types of additional networks, such as ipvlan, are not supported.
+
+[IMPORTANT]
+====
+Support for configuring multi-network policies for SR-IOV additional networks is a Technology Preview feature and:
```

Comment:
I think you are missing the word "is" at the end. Otherwise it reads "TP feature and only supported..."

I would just do sentences here for clarity:

"Support for configuring multi-network policies for SR-IOV additional networks is a Technology Preview feature and is only supported with kernel network interface cards (NICs). SR-IOV is not supported for Data Plane Development Kit (DPDK) applications."

This would be a great opportunity to `link:` to the [technology preview support scope](https://access.redhat.com/support/offerings/techpreview/).